app/lib/constants.json

```diff
@@ -0,0 +1 @@
+{ "metersInAMile": 1609 }
```

Comment:
I'd argue that the API should be metric and it is the responsibility of the client to do the necessary conversion.